project.clj

```diff
@@ -3,5 +3,9 @@
   :url "https://github.com/invetica/spec"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.9.0-alpha12"]
-                 [org.clojure/test.check "0.9.0" :scope "provided"]])
+  :dependencies [[org.clojure/clojure "1.9.0-alpha15"]
+                 [org.clojure/test.check "0.9.0" :scope "provided"]]
+  :aliases {"lint" ["do" ["whitespace-linter"] ["eastwood"]]}
+  :profiles
+  {:dev {:plugins [[jonase/eastwood "0.2.3"]
+                   [listora/whitespace-linter "0.1.0"]]}})
```

Comment:
I have a smaller version of the Whitespace Linter in the works. I should try to release that today.